client/src/includes/breadcrumbs.test.js

```diff
@@ -0,0 +1,70 @@
+import initCollapsibleBreadcrumbs from './breadcrumbs';
+
+describe('initCollapsibleBreadcrumbs', () => {
+  it('should do nothing if there is no breadcrumbs container', () => {
+    jest.spyOn(document, 'addEventListener').mockImplementation(() => {});
+    // Set up our document body
+    document.body.innerHTML =
+      '<div>' +
+      '  <span id="username" />' +
+      '  <button id="button" />' +
+      '</div>';
+    initCollapsibleBreadcrumbs();
+    // no event listeners registered
+    expect(document.addEventListener).not.toHaveBeenCalled();
+  });
+
+  describe('there is a breadcrumbs container present', () => {
+    jest.spyOn(document, 'addEventListener').mockImplementation(() => {});
```

Comment:
no need to `mockImplementation` - this will cause the `addEventListener` to do nothing and may cause your tests to work in unexpected ways.